docs/content/direct/common-setup-step-by-step.md

```diff
@@ -0,0 +1,71 @@
+# Common Setup - step-by-step
+
+The following steps create a deployment of KubeStellar Hub
+
+1. You may want to `set -e` in your shell so that any failures in the setup or usage scenarios are not lost.
+
+1. If you ran through these scenarios previously then you will need to do a bit of cleanup first. See how it is done in the cleanup script for our E2E tests (in `test/e2e/common/cleanup.sh`).
+
+1. Set environment variables to hold KubeStellar and OCM-status-addon desired versions:
+
+    ```shell
+    export KUBESTELLAR_VERSION=0.23.0-alpha.2
+    ```

```

Comment:
Don't these steps also belong in the alternative recipe using the hub chart?

Perhaps you could minimize how much this PR changes examples.md by leaving the current content in there but adding remarks like "The following steps will be replaced by [using the hub chart](common-setup-hub-chart.md)" before the to-be-replaced steps and following them with a remark like "Here end the steps to be replaced by [using the hub chart](common-setup-hub-chart.md)".